package.json

```diff
@@ -3,6 +3,10 @@
   "version": "0.0.1",
   "description": "Compound components for the Web",
   "type": "module",
+  "files": [
+    "./dist/**/*"
+  ],
```

Comment:
This was already taken care of by the `.npmignore` file: https://github.com/vector-im/compound-web/blob/main/.npmignore

In it I explicitly included the src/ folder and the tsconfig to have proper "Go to definition" working, alongside the typemaps.

One thing that might be worth excluding is the `src/res/` folder, which includes the fonts but is not really useful, since they are also in the `dist/` folder.

The current package looks like this (after running `yarn pack`):

```
drwxr-xr-x 0/0               0 2023-07-27 15:47 package
-rw-r--r-- 0/0           11357 2023-04-25 12:18 package/LICENSE
-rw-r--r-- 0/0            1434 2023-06-08 11:47 package/README.md
drwxr-xr-x 0/0               0 2023-07-27 15:47 package/dist
-rw-r--r-- 0/0            3382 2023-07-27 15:36 package/package.json
drwxr-xr-x 0/0               0 2023-07-27 13:31 package/src
-rw-r--r-- 0/0            1291 2023-07-27 15:26 package/tsconfig.json
-rw-r--r-- 0/0          143208 2023-07-27 15:47 package/dist/Inter-Bold.woff
-rw-r--r-- 0/0          106140 2023-07-27 15:47 package/dist/Inter-Bold.woff2
-rw-r--r-- 0/0          142552 2023-07-27 15:47 package/dist/Inter-Medium.woff
-rw-r--r-- 0/0          105924 2023-07-27 15:47 package/dist/Inter-Medium.woff2
-rw-r--r-- 0/0          133844 2023-07-27 15:47 package/dist/Inter-Regular.woff
-rw-r--r-- 0/0           98868 2023-07-27 15:47 package/dist/Inter-Regular.woff2
-rw-r--r-- 0/0          142932 2023-07-27 15:47 package/dist/Inter-SemiBold.woff
-rw-r--r-- 0/0          105804 2023-07-27 15:47 package/dist/Inter-SemiBold.woff2
-rw-r--r-- 0/0             852 2023-07-27 15:47 package/dist/check-circle.svg
-rw-r--r-- 0/0             559 2023-07-27 15:47 package/dist/close.svg
drwxr-xr-x 0/0               0 2023-07-27 15:47 package/dist/components
-rw-r--r-- 0/0             972 2023-07-27 15:47 package/dist/error.svg
-rw-r--r-- 0/0          144682 2023-07-27 15:47 package/dist/index.css
-rw-r--r-- 0/0             990 2023-07-27 15:47 package/dist/index.d.ts
-rw-r--r-- 0/0             940 2023-07-27 15:47 package/dist/index.d.ts.map
-rw-r--r-- 0/0           17569 2023-07-27 15:47 package/dist/index.js
-rw-r--r-- 0/0           43172 2023-07-27 15:47 package/dist/index.js.map
-rw-r--r-- 0/0             973 2023-07-27 15:47 package/dist/info.svg
-rw-r--r-- 0/0              40 2023-07-27 15:47 package/dist/setupTests.d.ts
-rw-r--r-- 0/0             159 2023-07-27 15:47 package/dist/setupTests.d.ts.map
drwxr-xr-x 0/0               0 2023-07-27 15:47 package/dist/utils
-rw-r--r-- 0/0            1222 2023-07-27 15:47 package/dist/visibility-invisible.svg
-rw-r--r-- 0/0            1002 2023-07-27 15:47 package/dist/visibility-visible.svg
drwxr-xr-x 0/0               0 2023-05-03 15:35 package/src/@types
drwxr-xr-x 0/0               0 2023-07-27 13:31 package/src/components
-rw-r--r-- 0/0            1666 2023-07-27 13:31 package/src/index.ts
drwxr-xr-x 0/0               0 2023-06-08 11:47 package/src/res
-rw-r--r-- 0/0            1068 2023-07-27 13:31 package/src/setupTests.ts
drwxr-xr-x 0/0               0 2023-06-08 11:47 package/src/styles
drwxr-xr-x 0/0               0 2023-06-08 11:47 package/src/utils
drwxr-xr-x 0/0               0 2023-07-27 15:47 package/dist/components/ActionControl
drwxr-xr-x 0/0               0 2023-07-27 15:47 package/dist/components/Alert
drwxr-xr-x 0/0               0 2023-07-27 15:47 package/dist/components/Avatar
drwxr-xr-x 0/0               0 2023-07-27 15:47 package/dist/components/Badge
drwxr-xr-x 0/0               0 2023-07-27 15:47 package/dist/components/Button
drwxr-xr-x 0/0               0 2023-07-27 15:47 package/dist/components/Checkbox
drwxr-xr-x 0/0               0 2023-07-27 15:47 package/dist/components/Form
drwxr-xr-x 0/0               0 2023-07-27 15:47 package/dist/components/Icon
drwxr-xr-x 0/0               0 2023-07-27 15:47 package/dist/components/Link
drwxr-xr-x 0/0               0 2023-07-27 15:47 package/dist/components/Radio
drwxr-xr-x 0/0               0 2023-07-27 15:47 package/dist/components/Toggle
drwxr-xr-x 0/0               0 2023-07-27 15:47 package/dist/components/Typography
-rw-r--r-- 0/0             262 2023-07-27 15:47 package/dist/utils/SuspenseImg.d.ts
-rw-r--r-- 0/0             358 2023-07-27 15:47 package/dist/utils/SuspenseImg.d.ts.map
drwxr-xr-x 0/0               0 2023-07-27 15:47 package/dist/utils/__ComponentTemplate__
-rw-r--r-- 0/0             383 2023-07-27 15:47 package/dist/utils/string.d.ts
-rw-r--r-- 0/0             279 2023-07-27 15:47 package/dist/utils/string.d.ts.map
-rw-r--r-- 0/0             745 2023-05-03 15:35 package/src/@types/custom.d.ts
drwxr-xr-x 0/0               0 2023-06-08 11:47 package/src/components/ActionControl
drwxr-xr-x 0/0               0 2023-06-08 11:47 package/src/components/Alert
drwxr-xr-x 0/0               0 2023-07-27 13:31 package/src/components/Avatar
drwxr-xr-x 0/0               0 2023-07-27 13:31 package/src/components/Badge
drwxr-xr-x 0/0               0 2023-07-27 15:25 package/src/components/Button
drwxr-xr-x 0/0               0 2023-06-08 11:47 package/src/components/Checkbox
drwxr-xr-x 0/0               0 2023-06-08 11:47 package/src/components/Form
drwxr-xr-x 0/0               0 2023-05-03 15:01 package/src/components/Icon
drwxr-xr-x 0/0               0 2023-06-08 11:47 package/src/components/Link
drwxr-xr-x 0/0               0 2023-06-08 11:47 package/src/components/Radio
drwxr-xr-x 0/0               0 2023-06-08 11:47 package/src/components/Toggle
drwxr-xr-x 0/0               0 2023-06-08 11:47 package/src/components/Typography
drwxr-xr-x 0/0               0 2023-06-08 11:47 package/src/res/fonts
-rw-r--r-- 0/0            2517 2023-06-08 11:47 package/src/styles/global.css
-rw-r--r-- 0/0            1952 2023-06-08 11:47 package/src/utils/SuspenseImg.tsx
drwxr-xr-x 0/0               0 2023-05-03 15:01 package/src/utils/__ComponentTemplate__
-rw-r--r-- 0/0            1233 2023-04-25 12:18 package/src/utils/string.ts
-rw-r--r-- 0/0             682 2023-07-27 15:47 package/dist/components/ActionControl/ActionControl.d.ts
-rw-r--r-- 0/0             674 2023-07-27 15:47 package/dist/components/ActionControl/ActionControl.d.ts.map
-rw-r--r-- 0/0            1171 2023-07-27 15:47 package/dist/components/ActionControl/ActionControl.stories.d.ts
-rw-r--r-- 0/0             263 2023-07-27 15:47 package/dist/components/ActionControl/ActionControl.stories.d.ts.map
-rw-r--r-- 0/0              59 2023-07-27 15:47 package/dist/components/ActionControl/ActionControl.test.d.ts
-rw-r--r-- 0/0             162 2023-07-27 15:47 package/dist/components/ActionControl/ActionControl.test.d.ts.map
-rw-r--r-- 0/0             389 2023-07-27 15:47 package/dist/components/Alert/Alert.d.ts
-rw-r--r-- 0/0             445 2023-07-27 15:47 package/dist/components/Alert/Alert.d.ts.map
-rw-r--r-- 0/0            1475 2023-07-27 15:47 package/dist/components/Alert/Alert.stories.d.ts
-rw-r--r-- 0/0             309 2023-07-27 15:47 package/dist/components/Alert/Alert.stories.d.ts.map
-rw-r--r-- 0/0              51 2023-07-27 15:47 package/dist/components/Alert/Alert.test.d.ts
-rw-r--r-- 0/0             138 2023-07-27 15:47 package/dist/components/Alert/Alert.test.d.ts.map
-rw-r--r-- 0/0             355 2023-07-27 15:47 package/dist/components/Avatar/Avatar.d.ts
-rw-r--r-- 0/0             448 2023-07-27 15:47 package/dist/components/Avatar/Avatar.d.ts.map
-rw-r--r-- 0/0            1605 2023-07-27 15:47 package/dist/components/Avatar/Avatar.stories.d.ts
-rw-r--r-- 0/0             321 2023-07-27 15:47 package/dist/components/Avatar/Avatar.stories.d.ts.map
-rw-r--r-- 0/0              52 2023-07-27 15:47 package/dist/components/Avatar/Avatar.test.d.ts
-rw-r--r-- 0/0             141 2023-07-27 15:47 package/dist/components/Avatar/Avatar.test.d.ts.map
-rw-r--r-- 0/0             443 2023-07-27 15:47 package/dist/components/Avatar/AvatarStack.d.ts
-rw-r--r-- 0/0             329 2023-07-27 15:47 package/dist/components/Avatar/AvatarStack.d.ts.map
-rw-r--r-- 0/0             511 2023-07-27 15:47 package/dist/components/Avatar/AvatarStack.stories.d.ts
-rw-r--r-- 0/0             241 2023-07-27 15:47 package/dist/components/Avatar/AvatarStack.stories.d.ts.map
-rw-r--r-- 0/0              57 2023-07-27 15:47 package/dist/components/Avatar/AvatarStack.test.d.ts
-rw-r--r-- 0/0             151 2023-07-27 15:47 package/dist/components/Avatar/AvatarStack.test.d.ts.map
-rw-r--r-- 0/0             319 2023-07-27 15:47 package/dist/components/Avatar/useIdColorHash.d.ts
-rw-r--r-- 0/0             213 2023-07-27 15:47 package/dist/components/Avatar/useIdColorHash.d.ts.map
-rw-r--r-- 0/0             304 2023-07-27 15:47 package/dist/components/Badge/Badge.d.ts
-rw-r--r-- 0/0             349 2023-07-27 15:47 package/dist/components/Badge/Badge.d.ts.map
-rw-r--r-- 0/0             617 2023-07-27 15:47 package/dist/components/Badge/Badge.stories.d.ts
-rw-r--r-- 0/0             229 2023-07-27 15:47 package/dist/components/Badge/Badge.stories.d.ts.map
-rw-r--r-- 0/0              51 2023-07-27 15:47 package/dist/components/Badge/Badge.test.d.ts
-rw-r--r-- 0/0             138 2023-07-27 15:47 package/dist/components/Badge/Badge.test.d.ts.map
-rw-r--r-- 0/0             500 2023-07-27 15:47 package/dist/components/Button/Button.d.ts
-rw-r--r-- 0/0             490 2023-07-27 15:47 package/dist/components/Button/Button.d.ts.map
-rw-r--r-- 0/0            2006 2023-07-27 15:47 package/dist/components/Button/Button.stories.d.ts
-rw-r--r-- 0/0             350 2023-07-27 15:47 package/dist/components/Button/Button.stories.d.ts.map
-rw-r--r-- 0/0              52 2023-07-27 15:47 package/dist/components/Button/Button.test.d.ts
-rw-r--r-- 0/0             141 2023-07-27 15:47 package/dist/components/Button/Button.test.d.ts.map
-rw-r--r-- 0/0             437 2023-07-27 15:47 package/dist/components/Checkbox/Checkbox.d.ts
-rw-r--r-- 0/0             481 2023-07-27 15:47 package/dist/components/Checkbox/Checkbox.d.ts.map
-rw-r--r-- 0/0            1435 2023-07-27 15:47 package/dist/components/Checkbox/Checkbox.stories.d.ts
-rw-r--r-- 0/0             277 2023-07-27 15:47 package/dist/components/Checkbox/Checkbox.stories.d.ts.map
-rw-r--r-- 0/0              54 2023-07-27 15:47 package/dist/components/Checkbox/Checkbox.test.d.ts
-rw-r--r-- 0/0             147 2023-07-27 15:47 package/dist/components/Checkbox/Checkbox.test.d.ts.map
-rw-r--r-- 0/0             490 2023-07-27 15:47 package/dist/components/Form/Control.d.ts
-rw-r--r-- 0/0             468 2023-07-27 15:47 package/dist/components/Form/Control.d.ts.map
drwxr-xr-x 0/0               0 2023-07-27 15:47 package/dist/components/Form/Controls
-rw-r--r-- 0/0             472 2023-07-27 15:47 package/dist/components/Form/Field.d.ts
-rw-r--r-- 0/0             464 2023-07-27 15:47 package/dist/components/Form/Field.d.ts.map
-rw-r--r-- 0/0             661 2023-07-27 15:47 package/dist/components/Form/Form.stories.d.ts
-rw-r--r-- 0/0             225 2023-07-27 15:47 package/dist/components/Form/Form.stories.d.ts.map
-rw-r--r-- 0/0             472 2023-07-27 15:47 package/dist/components/Form/Label.d.ts
-rw-r--r-- 0/0             464 2023-07-27 15:47 package/dist/components/Form/Label.d.ts.map
-rw-r--r-- 0/0             490 2023-07-27 15:47 package/dist/components/Form/Message.d.ts
-rw-r--r-- 0/0             468 2023-07-27 15:47 package/dist/components/Form/Message.d.ts.map
-rw-r--r-- 0/0             463 2023-07-27 15:47 package/dist/components/Form/Root.d.ts
-rw-r--r-- 0/0             461 2023-07-27 15:47 package/dist/components/Form/Root.d.ts.map
-rw-r--r-- 0/0             586 2023-07-27 15:47 package/dist/components/Form/Submit.d.ts
-rw-r--r-- 0/0             595 2023-07-27 15:47 package/dist/components/Form/Submit.d.ts.map
-rw-r--r-- 0/0             415 2023-07-27 15:47 package/dist/components/Form/ValidityState.d.ts
-rw-r--r-- 0/0             372 2023-07-27 15:47 package/dist/components/Form/ValidityState.d.ts.map
-rw-r--r-- 0/0             384 2023-07-27 15:47 package/dist/components/Form/index.d.ts
-rw-r--r-- 0/0             426 2023-07-27 15:47 package/dist/components/Form/index.d.ts.map
-rw-r--r-- 0/0             405 2023-07-27 15:47 package/dist/components/Icon/Icon.stories.d.ts
-rw-r--r-- 0/0             218 2023-07-27 15:47 package/dist/components/Icon/Icon.stories.d.ts.map
-rw-r--r-- 0/0             347 2023-07-27 15:47 package/dist/components/Link/Link.d.ts
-rw-r--r-- 0/0             398 2023-07-27 15:47 package/dist/components/Link/Link.d.ts.map
-rw-r--r-- 0/0             692 2023-07-27 15:47 package/dist/components/Link/Link.stories.d.ts
-rw-r--r-- 0/0             226 2023-07-27 15:47 package/dist/components/Link/Link.stories.d.ts.map
-rw-r--r-- 0/0              50 2023-07-27 15:47 package/dist/components/Link/Link.test.d.ts
-rw-r--r-- 0/0             135 2023-07-27 15:47 package/dist/components/Link/Link.test.d.ts.map
-rw-r--r-- 0/0             455 2023-07-27 15:47 package/dist/components/Radio/Radio.d.ts
-rw-r--r-- 0/0             482 2023-07-27 15:47 package/dist/components/Radio/Radio.d.ts.map
-rw-r--r-- 0/0            1432 2023-07-27 15:47 package/dist/components/Radio/Radio.stories.d.ts
-rw-r--r-- 0/0             268 2023-07-27 15:47 package/dist/components/Radio/Radio.stories.d.ts.map
-rw-r--r-- 0/0              51 2023-07-27 15:47 package/dist/components/Radio/Radio.test.d.ts
-rw-r--r-- 0/0             138 2023-07-27 15:47 package/dist/components/Radio/Radio.test.d.ts.map
-rw-r--r-- 0/0             563 2023-07-27 15:47 package/dist/components/Toggle/Toggle.d.ts
-rw-r--r-- 0/0             446 2023-07-27 15:47 package/dist/components/Toggle/Toggle.d.ts.map
-rw-r--r-- 0/0            1291 2023-07-27 15:47 package/dist/components/Toggle/Toggle.stories.d.ts
-rw-r--r-- 0/0             268 2023-07-27 15:47 package/dist/components/Toggle/Toggle.stories.d.ts.map
-rw-r--r-- 0/0              52 2023-07-27 15:47 package/dist/components/Toggle/Toggle.test.d.ts
-rw-r--r-- 0/0             141 2023-07-27 15:47 package/dist/components/Toggle/Toggle.test.d.ts.map
-rw-r--r-- 0/0             237 2023-07-27 15:47 package/dist/components/Typography/Body.d.ts
-rw-r--r-- 0/0             288 2023-07-27 15:47 package/dist/components/Typography/Body.d.ts.map
-rw-r--r-- 0/0             905 2023-07-27 15:47 package/dist/components/Typography/Body.stories.d.ts
-rw-r--r-- 0/0             236 2023-07-27 15:47 package/dist/components/Typography/Body.stories.d.ts.map
-rw-r--r-- 0/0              50 2023-07-27 15:47 package/dist/components/Typography/Body.test.d.ts
-rw-r--r-- 0/0             141 2023-07-27 15:47 package/dist/components/Typography/Body.test.d.ts.map
-rw-r--r-- 0/0             860 2023-07-27 15:47 package/dist/components/Typography/Heading.d.ts
-rw-r--r-- 0/0             667 2023-07-27 15:47 package/dist/components/Typography/Heading.d.ts.map
-rw-r--r-- 0/0             911 2023-07-27 15:47 package/dist/components/Typography/Heading.stories.d.ts
-rw-r--r-- 0/0             242 2023-07-27 15:47 package/dist/components/Typography/Heading.stories.d.ts.map
-rw-r--r-- 0/0              53 2023-07-27 15:47 package/dist/components/Typography/Heading.test.d.ts
-rw-r--r-- 0/0             147 2023-07-27 15:47 package/dist/components/Typography/Heading.test.d.ts.map
-rw-r--r-- 0/0             525 2023-07-27 15:47 package/dist/components/Typography/Typography.d.ts
-rw-r--r-- 0/0             521 2023-07-27 15:47 package/dist/components/Typography/Typography.d.ts.map
-rw-r--r-- 0/0             454 2023-07-27 15:47 package/dist/utils/__ComponentTemplate__/__ComponentTemplate__.d.ts
-rw-r--r-- 0/0             420 2023-07-27 15:47 package/dist/utils/__ComponentTemplate__/__ComponentTemplate__.d.ts.map
-rw-r--r-- 0/0             643 2023-07-27 15:47 package/dist/utils/__ComponentTemplate__/__ComponentTemplate__.stories.d.ts
-rw-r--r-- 0/0             272 2023-07-27 15:47 package/dist/utils/__ComponentTemplate__/__ComponentTemplate__.stories.d.ts.map
-rw-r--r-- 0/0              67 2023-07-27 15:47 package/dist/utils/__ComponentTemplate__/__ComponentTemplate__.test.d.ts
-rw-r--r-- 0/0             181 2023-07-27 15:47 package/dist/utils/__ComponentTemplate__/__ComponentTemplate__.test.d.ts.map
-rw-r--r-- 0/0             910 2023-06-08 11:47 package/src/components/ActionControl/ActionControl.module.css
-rw-r--r-- 0/0            1930 2023-06-08 11:47 package/src/components/ActionControl/ActionControl.tsx
-rw-r--r-- 0/0            1781 2023-06-08 11:47 package/src/components/Alert/Alert.module.css
-rw-r--r-- 0/0            2373 2023-06-08 11:47 package/src/components/Alert/Alert.tsx
-rw-r--r-- 0/0            3274 2023-07-27 13:31 package/src/components/Avatar/Avatar.module.css
-rw-r--r-- 0/0            1847 2023-07-27 13:31 package/src/components/Avatar/Avatar.tsx
-rw-r--r-- 0/0            2485 2023-07-27 13:31 package/src/components/Avatar/AvatarStack.tsx
-rw-r--r-- 0/0             896 2023-07-27 13:31 package/src/components/Avatar/avatar-clip.mask.svg
-rw-r--r-- 0/0            1067 2023-07-27 13:31 package/src/components/Avatar/useIdColorHash.ts
-rw-r--r-- 0/0            1047 2023-07-27 13:31 package/src/components/Badge/Badge.module.css
-rw-r--r-- 0/0            1179 2023-07-27 13:31 package/src/components/Badge/Badge.tsx
-rw-r--r-- 0/0            3615 2023-06-08 11:47 package/src/components/Button/Button.module.css
-rw-r--r-- 0/0            1592 2023-06-08 11:47 package/src/components/Button/Button.tsx
-rw-r--r-- 0/0            2957 2023-06-08 11:47 package/src/components/Checkbox/Checkbox.module.css
-rw-r--r-- 0/0            1675 2023-06-08 11:47 package/src/components/Checkbox/Checkbox.tsx
-rw-r--r-- 0/0            1248 2023-06-08 11:47 package/src/components/Form/Control.tsx
drwxr-xr-x 0/0               0 2023-06-08 11:47 package/src/components/Form/Controls
-rw-r--r-- 0/0            1226 2023-06-08 11:47 package/src/components/Form/Field.tsx
-rw-r--r-- 0/0            1226 2023-06-08 11:47 package/src/components/Form/Label.tsx
-rw-r--r-- 0/0            1373 2023-06-08 11:47 package/src/components/Form/Message.tsx
-rw-r--r-- 0/0            1215 2023-06-08 11:47 package/src/components/Form/Root.tsx
-rw-r--r-- 0/0            1409 2023-06-08 11:47 package/src/components/Form/Submit.tsx
-rw-r--r-- 0/0            1008 2023-06-08 11:47 package/src/components/Form/ValidityState.tsx
-rw-r--r-- 0/0            2319 2023-06-08 11:47 package/src/components/Form/form.module.css
-rw-r--r-- 0/0             913 2023-04-25 12:18 package/src/components/Form/index.ts
drwxr-xr-x 0/0               0 2023-04-25 12:18 package/src/components/Icon/icons
-rw-r--r-- 0/0            1345 2023-06-08 11:47 package/src/components/Link/Link.module.css
-rw-r--r-- 0/0            1118 2023-06-08 11:47 package/src/components/Link/Link.tsx
-rw-r--r-- 0/0            2961 2023-06-08 11:47 package/src/components/Radio/Radio.module.css
-rw-r--r-- 0/0            1520 2023-06-08 11:47 package/src/components/Radio/Radio.tsx
-rw-r--r-- 0/0            2588 2023-06-08 11:47 package/src/components/Toggle/Toggle.module.css
-rw-r--r-- 0/0            1604 2023-06-08 11:47 package/src/components/Toggle/Toggle.tsx
-rw-r--r-- 0/0             874 2023-04-25 12:18 package/src/components/Typography/Body.tsx
-rw-r--r-- 0/0            2047 2023-06-08 11:47 package/src/components/Typography/Heading.tsx
-rw-r--r-- 0/0            1372 2023-04-25 12:18 package/src/components/Typography/Typography.tsx
drwxr-xr-x 0/0               0 2023-06-08 11:47 package/src/res/fonts/Inter
-rw-r--r-- 0/0             673 2023-04-25 12:18 package/src/utils/__ComponentTemplate__/__ComponentTemplate__.module.css
-rw-r--r-- 0/0            1288 2023-04-25 12:18 package/src/utils/__ComponentTemplate__/__ComponentTemplate__.tsx
drwxr-xr-x 0/0               0 2023-07-27 15:47 package/dist/components/Form/Controls/MFA
drwxr-xr-x 0/0               0 2023-07-27 15:47 package/dist/components/Form/Controls/Password
drwxr-xr-x 0/0               0 2023-06-08 11:47 package/src/components/Form/Controls/MFA
drwxr-xr-x 0/0               0 2023-06-08 11:47 package/src/components/Form/Controls/Password
-rw-r--r-- 0/0             573 2023-04-25 12:18 package/src/components/Icon/icons/thread.svg
-rw-r--r-- 0/0          143208 2023-06-08 11:47 package/src/res/fonts/Inter/Inter-Bold.woff
-rw-r--r-- 0/0          106140 2023-06-08 11:47 package/src/res/fonts/Inter/Inter-Bold.woff2
-rw-r--r-- 0/0          142552 2023-06-08 11:47 package/src/res/fonts/Inter/Inter-Medium.woff
-rw-r--r-- 0/0          105924 2023-06-08 11:47 package/src/res/fonts/Inter/Inter-Medium.woff2
-rw-r--r-- 0/0          133844 2023-06-08 11:47 package/src/res/fonts/Inter/Inter-Regular.woff
-rw-r--r-- 0/0           98868 2023-06-08 11:47 package/src/res/fonts/Inter/Inter-Regular.woff2
-rw-r--r-- 0/0          142932 2023-06-08 11:47 package/src/res/fonts/Inter/Inter-SemiBold.woff
-rw-r--r-- 0/0          105804 2023-06-08 11:47 package/src/res/fonts/Inter/Inter-SemiBold.woff2
-rw-r--r-- 0/0             516 2023-07-27 15:47 package/dist/components/Form/Controls/MFA/MFA.d.ts
-rw-r--r-- 0/0             454 2023-07-27 15:47 package/dist/components/Form/Controls/MFA/MFA.d.ts.map
-rw-r--r-- 0/0              49 2023-07-27 15:47 package/dist/components/Form/Controls/MFA/MFA.test.d.ts
-rw-r--r-- 0/0             152 2023-07-27 15:47 package/dist/components/Form/Controls/MFA/MFA.test.d.ts.map
-rw-r--r-- 0/0              71 2023-07-27 15:47 package/dist/components/Form/Controls/MFA/index.d.ts
-rw-r--r-- 0/0             185 2023-07-27 15:47 package/dist/components/Form/Controls/MFA/index.d.ts.map
-rw-r--r-- 0/0             379 2023-07-27 15:47 package/dist/components/Form/Controls/Password/Password.d.ts
-rw-r--r-- 0/0             386 2023-07-27 15:47 package/dist/components/Form/Controls/Password/Password.d.ts.map
-rw-r--r-- 0/0              54 2023-07-27 15:47 package/dist/components/Form/Controls/Password/Password.test.d.ts
-rw-r--r-- 0/0             167 2023-07-27 15:47 package/dist/components/Form/Controls/Password/Password.test.d.ts.map
-rw-r--r-- 0/0              81 2023-07-27 15:47 package/dist/components/Form/Controls/Password/index.d.ts
-rw-r--r-- 0/0             190 2023-07-27 15:47 package/dist/components/Form/Controls/Password/index.d.ts.map
-rw-r--r-- 0/0             888 2023-06-08 11:47 package/src/components/Form/Controls/MFA/MFA.module.css
-rw-r--r-- 0/0            2381 2023-06-08 11:47 package/src/components/Form/Controls/MFA/MFA.tsx
-rw-r--r-- 0/0             599 2023-06-08 11:47 package/src/components/Form/Controls/MFA/index.ts
-rw-r--r-- 0/0            1794 2023-06-08 11:47 package/src/components/Form/Controls/Password/Password.tsx
-rw-r--r-- 0/0             609 2023-06-08 11:47 package/src/components/Form/Controls/Password/index.ts
```